index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trail-Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+        <div id="page-wrapper">
+          <div id="header">
+            <div class="logo">
+              <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo">
+            </div>
+    
+            <div id="nav-bar">
+              <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+              </ul>
+            </div>
+          </div>
```

Comment:
- I noticed that in your code, you're using the <div> tag instead of a semantic tag that could better describe the content. Semantic HTML provides more meaning and context to web pages, which can be helpful for accessibility and search engine optimization.